smartsim/_core/launcher/step/alpsStep.py

```diff
@@ -49,13 +49,29 @@ def __init__(self, name: str, cwd: str, run_settings: AprunSettings) -> None:
         :type run_settings: AprunSettings
         """
         super().__init__(name, cwd, run_settings)
-        self.alloc = None
+        self.alloc: t.Optional[str] = None
         if not self.run_settings.in_batch:
             self._set_alloc()
 
     @property
-    def run_settings(self) -> AprunSettings:
-        return self.step_settings
+    def run_settings(self) -> RunSettings:
```

Comment:
Do you think we should have a concise docstring for this kind of `@property` (mostly for the rendered docs)?

Reply:
Added `"""Get the run settings attached to this step"""`